lib/validators/movie.js

```diff
@@ -1,8 +0,0 @@
-'use strict';
-
-const Joi = require('joi');
-
-module.exports = Joi.object().keys({
-  title: Joi.string().min(1).max(255).required(),
-  release_year: Joi.number().integer().min(1878).max(9999).optional()
-});
```

Comment:
i moved this under lib/validators/movies/create.js earlier but had forgotten to remove this file. So removing it now.